src/Common/src/TypeSystem/Ecma/MetadataExtensions.cs

```diff
@@ -72,6 +90,25 @@ public static class MetadataExtensions
             return default(CustomAttributeHandle);
         }
 
+        public static IEnumerable<CustomAttributeHandle> GetCustomAttributeHandles(this MetadataReader metadataReader, CustomAttributeHandleCollection customAttributes,
```

Comment:
Nit: It might be better to inline this into `GetDecodedCustomAttributes`. We tend to keep metadata helpers as close to 0 allocations as possible (because the underlying API is pretty much 0 allocations by design). With `GetDecodedCustomAttributes` calling into this, the code is allocating two enumerators. I (as a caller) would not expect the API to do that.